### DISCUSSION TITLE
The importance of defaults

### DISCUSSION THREAD

**Discussion by ray1claw** [elementary/edw#47] [Withdrawn or Not Accepted]
I would want to give a talk about how crucial a default out-of-the-box experience is to a user. How the default experience must give the user what they need before taking them on a journey where the user can find complexity and advancement at their own pace.

It's a super vague idea right now, but it's something very important when crafting any UX.

**Comment by cassidyjames** [comment]
@ray1claw I like this, as it jives with the [Accessible Configuration](https://docs.elementary.io/hig/design-philosophy/accessible-configuration) aspect of our design philosophy. I'd highly recommend using the HIG to help with some direction in this talk, and perhaps we could even update the HIG based on how you communicate it. :smile:

**Comment by cassidyjames** [comment]
@ray1claw  I'm not sure if you saw, but we've accepted your talk. :) You can get started writing it now if you'd like, and we will provide further instructions/guidance around the recording once we've accepted all the talks.

**Reply by ray1claw** [reply]
Hey I somehow missed all notifications and am seeing this now only thanks to your email 😓

Due to some other commitments and short time left with no prep so far, would like to forego this one please. Hope that's okay.

**Reply by cassidyjames** [reply]
@ray1claw not a problem, thanks for getting back to us.
